pkg/customresourcestate/config.go

```diff
@@ -166,8 +165,12 @@ type ConfigDecoder interface {
 	Decode(v interface{}) (err error)
 }
 
+type GVKToGVKP interface {
```

Comment:
```suggestion
// GVKToGVKP incorporates logic to resolve a given GVK, with variable version and/or kind, to it's []discovery.GroupVersionKindPlural representation.
type GVKToGVKP interface {
```

Looks like we need to annotate this, so the linter is happy.